cw/cw305/util/spiflash.py

```diff
@@ -0,0 +1,27 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+r"""Utilities used to download firmware images into the OpenTitan system."""
+
+import subprocess
+
+class FtdiProgrammer(object):
+  """Spiflash executable wrapper.
+
+  See doc/getting_started.md for details on how to connect the FTDI cable to
+  the CW305 FPGA target board.
+  """
+  def __init__(self, spiflash_path, dev_id, dev_sn, input):
+    """Initializes spiflash FTDI programmer.
+
+    @param spiflash_path Path to spiflash executable.
+    @param dev_id FTDI USB device ID in vendor:product hex format.
+    @param dev_sn FTDI USB serial number as reported by lsusb.
+    @param input Path to OpenTitan firmware image.
```

Comment:
Are we using doxygen for python as well? If not, we should probably follow [this](https://google.github.io/styleguide/pyguide.html#383-functions-and-methods).

Reply:
SGTM. I prefer the format you suggested.

I will let lowRISC weigh on this for additional coding style guidelines across projects.